docs/config.example.yaml

```diff
@@ -68,3 +67,10 @@ votera:
 
     # Endpoint of Agora
     votera_endpoint: http://dev-votera.bosagora.com:1337/
+
+################################################################################
+##                               Node server config                           ##
+################################################################################
+node:
+    # Endpoint of Agora
+    node_endpoint: https://v2.bosagora.io/
```

Comment:
agora_endpoint: http://127.1.1.1:4567
You can refer to this without adding anything new.